docs/get-started/using-standalone.md

```diff
@@ -0,0 +1,110 @@
+# Using deck.gl without React (Experimental)
```

Comment:
Nit: This heading is same as following sub-heading, may be change this to `Using deck.gl standalone` ?

Reply:
Good catch. Updated second heading to say "Example Code"